### DISCUSSION TITLE
Mitsubishi Heavy Industries - ESP8266 DOUT/DIN Pins

### DISCUSSION THREAD

**Discussion by Meesen0743** [Arnold-n/P1P2MQTT#53] [Q&A] [answered]
Hi everyone, 

A long time ago I started a repo to interface with my Mitsubishi Heavy Industries (MHI) HVAC Specifically "Fdua100vf2" (X,Y Bus) but unfortunately I have had no time to make any progress due to work and the lack of programming knowledge. 

I am currently trying to configure your ESP8266 example to work with a board I have had made but due to my lack of coding knowledge and my limited ability to find where I can define pins associated with the ESP8266 I cannot progress.

The PCB I have incorporates the Max22088 with the following configuration;

DOUT - Pin 15 (Hardware Serial) OR Pin 5 (D6)
DIN - Pin 16 (Hardware Serial) OR Pin 6 (D7)

Once this is established I am hoping to then start interpreting data received. 

Any help is appreciated :)

P.s I am also open to paying for someone's time to help me make progress, So please contact me If this is something you may be interested in.

**Comment by Arnold-n** [comment, marked as answer]
Hi @Meesen0743,
This project needs the MAX22088 to be connected to an ATmega328P, which runs P1P2Monitor, which then connects via serial to the ESP8266. The reason for that is that - assuming MHI also uses the Home Bus System - the MHI is not a true serial interface.
